<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="barEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="AdultspinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="BigCspinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="SmallCspinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>AIKUISIA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ISOJA LAPSIA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PIENIÃ„ LAPSIA:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_Button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>VIIVAKOODI :</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
